### DISCUSSION TITLE
Don't get the 'manager' option in menu

### DISCUSSION THREAD

**Discussion by fizion** [Comfy-Org/ComfyUI-Manager#39] [Q&A]
Hi!

I've followed the steps for installation, and it says ComfyUI-Manager is loaded in 0.3 seconds so seems installation was fine. But for me the bottom option of the menu is still 'Load Default' and not 'Manager'. But all files seem to be in the right place as well.

I don't see more steps I should do so not sure what I can try. Could it be caused by the fact I don't have custom nodes yet in the folder?

If someone can kick me in the right direction, please!

**Comment by ltdrdata** [comment]
> Hi!
> 
> I've followed the steps for installation, and it says ComfyUI-Manager is loaded in 0.3 seconds so seems installation was fine. But for me the bottom option of the menu is still 'Load Default' and not 'Manager'. But all files seem to be in the right place as well.
> 
> I don't see more steps I should do so not sure what I can try. Could it be caused by the fact I don't have custom nodes yet in the folder?
> 
> If someone can kick me in the right direction, please!

Are you still experiencing the same symptoms even after refreshing the browser following the installation? If there are no error messages like "import failed" in the terminal, please open the developer tools (F12) in your browser (Chrome, for instance) and check the console messages.

**Reply by fizion** [reply]
Thanks for replying! Still issues, I thought it might be some cache issue but don't see anything off actually. No errors, also a clean console and everything seems to be at the right place. 

A few times I just deleted the folder and tried to install again, both with the .bat and with the command line stuff. No errors on install as well. If I can check some other things, let me know, more than happy too. I also checked the network tab in dev tools but besides a favicon everything is loading. I don't have to add something to the _**run_nvidia_gpu.bat**_, if so I totally missed it.

Just tried this but still same: _git update-ref refs/remotes/origin/main a361cc1 && git fetch --all && git pull_

![comfy](https://github.com/ltdrdata/ComfyUI-Manager/assets/4068160/8cfd7759-7fc4-4ac6-bc10-401b60c295ea)

**Comment by ltdrdata** [comment]
> Thanks for replying! Still issues, I thought it might be some cache issue but don't see anything off actually. No errors, also a clean console and everything seems to be at the right place.
> 
> A few times I just deleted the folder and tried to install again, both with the .bat and with the command line stuff. No errors on install as well. If I can check some other things, let me know, more than happy too. I also checked the network tab in dev tools but besides a favicon everything is loading. I don't have to add something to the _**run_nvidia_gpu.bat**_, if so I totally missed it.
> 
> Just tried this but still same: _git update-ref refs/remotes/origin/main [a361cc1](https://github.com/ltdrdata/ComfyUI-Manager/commit/a361cc17f515fa629e703441a09c415112716ac3) && git fetch --all && git pull_
> 
> ![comfy](https://user-images.githubusercontent.com/4068160/257068255-8cfd7759-7fc4-4ac6-bc10-401b60c295ea.png)

Your ComfyUI seems to be in a very unusual state. PrimitiveNode and Note nodes should not appear as missing nodes since they are built-in nodes. There might be a conflict with other custom nodes causing this issue.

Try removing all web extensions in web/extensions except for the web/extensions/core.

Also, since custom nodes can be reinstalled at runtime, it is recommended to move all custom nodes except for ComfyUI-Manager to a different location temporarily and disable them for testing.

**Reply by fizion** [reply]
Well that explains a lot, I wrote also the guy who made the node scheme I used and asked where I could download those custom nodes. He said he liked to use only default nodes. So I thought it was translation to Chinese gone wrong but this explains more. Very odd, I don't have any custom nodes so I'll have to puzzle a bit. If I find what caused it I will report back, might be useful for others.

**Comment by fizion** [comment]
Hi, I want you to know the issue is solved. As you see in the screenshot ComfyUI is placed in this directory "[AI]". I use those brackets quite a lot since those will be on top of the list. After some fresh copies of ComfyUI and still running into the issue about missing nodes and the node manager not appearing I moved it to the root directory.

All works now. I think this might be a bug with ComfyUI since other AI's work well from that directory but I'm happy I've solved it. I did also reported an issue on the ComfyUI github, might be useful for them.

Strange thing was that the default node setup worked perfectly fine. But everything else didn't basically.

Thanks again for your time!
